app/models/feedback_submission.rb

```diff
@@ -59,19 +63,22 @@ def submit
       return @errors.blank?
   end
 
+  def jira_issue(jira_config)
+    JIRA::Client.new({
+      site: jira_config['jira_url'],
+      context_path: '/',
+      username: jira_config['username'],
+      password: jira_config['password'],
+      auth_type: :basic
+    }).Issue.build
+  end
+
   def submit_to_jira
     jira_config = @form_config['target']['jira']
 
     begin
-      client = JIRA::Client.new({
-        :site => jira_config['jira_url'],
-        :context_path => '/',
-        :username => jira_config['username'],
-        :password => jira_config['password'],
-        :auth_type => :basic
-      })
+      issue = jira_issue(jira_config)
```

Comment:
```suggestion
      issue = build_jira_issue(jira_config)
```
(related to above comment)